doc/source/code-examples/advancedexamples.rst

```diff
@@ -145,6 +145,63 @@ however the user may create the full state as follows:
     # ``final_state`` is a ``tf.Tensor``
 
 
+How to select a quantum hardware device for circuit execution?
+--------------------------------------------------------------
+
+Qibolab is the dedicated Qibo backend for quantum hardware control.
+For installation instructions, see the `Qibolab Documentation <https://qibo.science/qibolab/stable/>`_.
+The ``Platform`` class in Qibolab represents a QPU device controlled by one or more instruments.
+By specifying the platform name, the user can select the quantum hardware device for circuit execution.
+When executing the circuit, it will be automatically transpiled using the :ref:`Default Transpiler <tutorials_set_transpiler>`.
+
+.. code-block:: python
+
+    import qibo
+
+    # Set the backend and platform
+    qibo.set_backend("qibolab", platform="qw5q_platinum")
+
+How to select specific hardware qubits for circuit execution?
+-------------------------------------------------------------
+
+The :class:`qibo.models.Circuit` has a ``wire_names`` property that stores the physical names of the qubits in the circuit.
+The physical qubit name ``wire_names[i]`` is assigned to the ``i`` th qubit in the circuit.
+Users can specify the hardware qubits to be used by setting the ``wire_names``.
+During circuit execution, Qibolab uses the ``wire_names`` to find the corresponding hardware qubits.
+
+
+.. code-block:: python
+
+    from qibo import Circuit, gates
+
+    # Define the circuit with 4 qubits
+    circuit = Circuit(4)
+    circuit.add(gates.H(0))
+    circuit.add(gates.CNOT(0, 1))
+    circuit.add(gates.CZ(0, 2))
+    circuit.add(gates.M(2, 3))
+
+    # Set the physical qubit names
+    circuit.wire_names = ["C", "A", "B", "D"]
+
+    # This is equivalent to:
+    # H gate on qubit "C"
+    # CNOT gate on qubits "C" and "A"
+    # CZ gate on qubits "C" and "B"
+    # M gate on qubit "B" and "D"
```

Comment:
Explicitly say that this is equal to doing `Circuit(4, wire_names=[...])`

Reply:
[8e903f0](https://github.com/qiboteam/qibo/pull/1531/commits/8e903f02983393a18e42d29a32165f053c33c7d9)